APIs/schemas/receiver_transport_params_mqtt.json

```diff
@@ -53,6 +53,18 @@
               "secure-mqtt"
             ]
           },
+          "broker_authorization": {
+            "type": [
+              "string",
+              "boolean"
+            ],
+            "description": "Indication of whether authorization is used for communication with the broker. If the parameter is set to auto the Receiver should establish for itself whether authorization should be used, based on a discovery mechanism or its own internal configuration. ",
```

Comment:
Pedant mode: there's an extra space on the end of the description string!

Reply:
Some people may like `"auto"` for this to mean follow-this-API-authorization-setting (and similar logic when `"auto"` is specified for `"broker_protocol"`) but I think that's covered by 'internal configuration'?